cluster/src/main/java/org/apache/iotdb/cluster/config/ClusterConfig.java

```diff
@@ -127,6 +125,34 @@
    */
   private int readDataConsistencyLevel = 1;
 
+  /**
+   * Max num of threads can concurrently execute qp sub-tasks of all client requests. When <= 0, use
+   * CPU core number * 10. Each client request corresponds to a QP Task. A QP task may be divided
+   * into several sub-tasks.
+   */
+  private int concurrentQPSubTaskThread = Runtime.getRuntime().availableProcessors() * 10;
+
+  /**
+   * Max num of threads can concurrently apply tasks in data partition state machine. When <= 0, use
+   * CPU core number.Raft Task refers to tasks that need to be performed in the state machine.
+   */
+  private int concurrentRaftTaskThread = Runtime.getRuntime().availableProcessors();
+
+  /**
+   * Max time of blocking main thread for waiting for all running task threads and tasks in the
+   * queue until end. Raft Task refers to tasks that need to be performed in the state machine.The
+   * unit is milliseconds. Due to guarantee data consistency, it must ensure that tasks finish. If
+   * user sets the parameter, it will not sure to guarantee consistency.
+   */
+  private int closeRaftTaskBlockTimeout = Integer.MAX_VALUE;
+
+  /**
+   * Max time of blocking main thread for waiting for all running task threads and tasks in the
+   * queue until end. Each client request corresponds to a QP Task. A QP task may be divided into
+   * several sub-tasks.The unit is milliseconds.
+   */
+  private int closeQPSubTaskBlockTimeout = 1000;
```

Comment:
When a node receives a request from client, if it finds itself unable to process the request, 
then it sends requests to other nodes in the cluster. 
This parameter represents the maximum timeout for these requests.
The unit is milliseconds.

Reply:
I think you may misunderstand the meaning of this parameter. What you describe is task_timeout_ms

Reply:
So you should rewrite this comment to make it more readable

Reply:
It may not be necessary for users to set such detailed system operating parameters.